src/NHSD.BuyingCatalogue.Identity.Api/Controllers/ConsentController.cs

```diff
@@ -46,5 +52,17 @@ public async Task<IActionResult> Index(ConsentViewModel model)
 
             return View("Error");
         }
+
+        [AllowAnonymous]
```

Comment:
There's currently no authorization applied to `ConsentController` so the `AllowAnonymous` attribute shouldn't be needed.

Reply:
removed